docs/content/guides/reverse-proxy.md

```diff
@@ -5,6 +5,11 @@ configs that you'll have to do.
 
 This documentation will cover HTTPS setup, with comments for HTTP setup.
 
+## Cloudflare
+!!! warning "Requirements on your server"
+    If you use Cloudflare as reverse proxy then you **MUST** disable the minify features for HTML, CSS and JS, or your HedgeDoc instance may be not inaccessible or broken.
```

Comment:
```suggestion
    If you use Cloudflare as reverse proxy then you **MUST** disable the minify features for HTML, CSS and JS, or your HedgeDoc instance may be not accessible or broken.
```

either `not accessible` or `inaccessible`

Reply:
Good point. It was "not available" first, but I like this term more.